src/essreflectometry/amor/types.py

```diff
@@ -0,0 +1,54 @@
+from typing import NewType, TypeVar
+
+import sciline
+import scipp as sc
+
+from ..reflectometry.types import Run
+
+# TODO What do they mean?
+# Supermirror parameters
+MValue = NewType('MValue', str)
+CriticalEdge = NewType('CriticalEdge', str)
+Alpha = NewType('Alpha', str)
+SupermirrorParameter = TypeVar('SupermirrorParameter', MValue, CriticalEdge, Alpha)
+
+
+class Supermirror(sciline.Scope[SupermirrorParameter, sc.Variable], sc.Variable):
+    """Supermirror parameter scope."""
+
+
+class SampleRotation(sciline.Scope[Run, sc.Variable], sc.Variable):
+    """The rotation of the sample / the reference sample."""
+
+
+class BeamSize(sciline.Scope[Run, sc.Variable], sc.Variable):
+    """FWHM of the neutron beam."""
+
+
+class DetectorSpatialResolution(sciline.Scope[Run, sc.Variable], sc.Variable):
+    # TODO
+    """Don't know what this is."""
```

Comment:
Roughly the pixel-to-pixel spacing, I suppose.

Reply:
Probably, but I can't write that in the docstring! ;)
I've created an issue for defining the pipeline parameters.